### PR TITLE
clean up remove local dev env set up

### PR DESCRIPTION
**Pull request by SamYuan1990**
so far this PR will break as local-dev-env is not ready...

Signed-off-by: Sam Yuan <yy19902439@126.com>

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -35,8 +35,8 @@ async function kubectl() {
 
 async function kind() {
   core.info(`kind`);
-  shell.exec("git clone https://github.com/sustainable-computing-io/KeplerK8SAction --depth=1");
-  shell.exec("cd KeplerK8SAction && bash -c ./kind/common.sh")
+  shell.exec("git clone https://github.com/sustainable-computing-io/local-dev-cluster.git --depth=1");
+  shell.exec("cd local-dev-cluster && bash -c ./mian.sh")
```
Comment (SamYuan1990) [inline index.js:39]: @rootfs , @marceloamaral if we use https://github.com/sustainable-computing-io/local-dev-cluster to save our local env set up code, but not this repo. this line will skip testing... as the code is out of CI scope.
if we want to cover into CI testing, https://github.com/sustainable-computing-io/kepler-action/pull/19/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R39 this PR is a sample.

Comment (rootfs) [reply index.js]: Looks good, let's go this path!

File: index.js [hunk 2]
```diff
@@ -35,8 +35,8 @@ async function kubectl() {
 
 async function kind() {
   core.info(`kind`);
-  shell.exec("git clone https://github.com/sustainable-computing-io/KeplerK8SAction --depth=1");
-  shell.exec("cd KeplerK8SAction && bash -c ./kind/common.sh")
+  shell.exec("git clone https://github.com/sustainable-computing-io/local-dev-cluster.git --depth=1");
+  shell.exec("cd local-dev-cluster && bash -c ./main.sh")
```
[the remark this answers is not in the document]
Comment (SamYuan1990) [reply index.js:39]: @rootfs , the question is, if we moving local dev set up to local-dev-cluster, for CI in this repo, do we need to verify cluster set up?

[the remark this answers is not in the document]
Comment (rootfs) [reply index.js:39]: no, that validation should happen in the `local-cluster-dev`

**Review by rootfs — APPROVED** [review]

**Pull request merged**